push.go

```diff
@@ -682,3 +690,37 @@ func (localeFile *LocaleFile) shouldCreateLocale(source *Source) bool {
 	// every other source should be uploaded and validated in uploads#create
 	return (localeFile.Name != "" || localeFile.Code != "")
 }
+
+func (source *Source) localeShow(client *phraseapp.Client, localeFile *LocaleFile) (*phraseapp.LocaleDetails, error) {
```

Comment:
I would add a third parameter `found bool` which we can use to verify if something was found